src/NzbDrone.Core/Download/Clients/Deluge/DelugeSettings.cs

```diff
@@ -44,7 +44,7 @@ public DelugeSettings()
         [FieldDefinition(4, Label = "Password", Type = FieldType.Password, Privacy = PrivacyLevel.Password)]
         public string Password { get; set; }
 
-        [FieldDefinition(5, Label = "Category", Type = FieldType.Textbox, HelpText = "Adding a category specific to Sonarr avoids conflicts with unrelated downloads, but it's optional")]
+        [FieldDefinition(5, Label = "Category", Type = FieldType.Textbox, HelpText = "Adding a category specific to Sonarr avoids conflicts with unrelated downloads")]
```

Comment:
Why though? It's still optional.

Reply:
keeping it in means users then are confused why they have random downloads showing up given showunknown is enabled which solved the opposite problem of downloads 'disappearing' 

I guess `avoids conflicts` is not clear enough?

I think regardless it does need some sort of reword

Reply:
Maybe: `Adding a category specific to Sonarr avoids conflicts with non-TV Series downloads. Using a category is optional, but strongly recommended.` still optional, but now recommended.

Reply:
pushed as `Adding a category specific to Sonarr avoids conflicts with unrelated non-Sonarr downloads. Using a category is optional, but strongly recommended.`